packages/api/src/controllers/generate.test.ts

```diff
@@ -71,6 +71,16 @@ afterEach(async () => {
   await clearDatabase(server);
 });
 
+const testBothRoutes = (testFn) => {
+  describe("generate route", () => {
+    testFn("/generate");
+  });
+
+  describe("beta generate route", () => {
+    testFn("/beta/generate");
+  });
+};
```

Comment:
Keep in mind we need to update CloudFlare routes to also skip the regular worker on the new route.